firebase-debug.log

```diff
@@ -0,0 +1,66 @@
+[debug] [2019-07-29T20:50:45.874Z] ----------------------------------------------------------------------
```

Comment:
Can we add firebase-debug.log to our gitignore?

Reply:
Yes I'll do that.